BridgeAppSDK/SBAProfileDataSource.swift

```diff
@@ -33,6 +33,9 @@
 
 import Foundation
 
+public var SBAProfileJSONFilename = "Profile"
+public var SBAProfileDataSourceClassType = "ProfileDataSource"
```

Comment:
Why `var` rather than `let`? I'm not actually sure how public vars that aren't in a class work.